README.md

```diff
@@ -309,13 +310,27 @@ A certificate may be supplied in the manner described above and can be used to p
 
 Here is an example of a Kubernetes secret bearing a certificate for use with a specific fully-qualified domain name.  The following criteria must be met:
 
-* Must be named `<domain>-cert`
+* Secret name must be for the form `<arbitrary name>-cert`
+  * This must be associated to the domain using the [router.deis.io/certificates](#certificates-annotation) annotation.
```

Comment:
That was deliberate.  The second bullet is indented from the one above to provide supplementary information on how a cert that's named arbitrarily is associated to a domain.